### ISSUE TITLE
TimeInput - name attribute accessibility violation WCAG 1.1.1

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#6566]
An input should have a name associated with it. Currently the name is getting put on the hidden input within TimeInput. It should instead be placed on the regular input so that it is available to screen reader users.

Related WCAG Rule
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html
